pkg/services/alerting/conditions/query.go

```diff
@@ -109,9 +109,17 @@ func (c *QueryCondition) executeQuery(context *alerting.EvalContext, timeRange *
 		return nil, fmt.Errorf("Could not find datasource %v", err)
 	}
 
-	req := c.getRequestForAlertRule(getDsInfo.Result, timeRange)
+	req := c.getRequestForAlertRule(getDsInfo.Result, timeRange, context.IsDebug)
 	result := make(tsdb.TimeSeriesSlice, 0)
 
+	if context.IsDebug {
+		data := simplejson.NewFromAny(req)
```

Comment:
Should probable convert req to dto version or specific struct of request so that we don't return all properties of datasource like sensitive information.

Reply:
Changed the query model to return only datasource id and name:
![image](https://user-images.githubusercontent.com/1668778/55627950-b93a2680-57af-11e9-91a7-9c0d6580123b.png)